src/client/Microsoft.Identity.Client/Platforms/Android/EmbeddedWebview/AuthenticationAgentActivity.cs

```diff
@@ -111,6 +113,24 @@ public override bool ShouldOverrideUrlLoading(WebView view, string url)
                     return true;
                 }
 
+                if (url.StartsWith(BrokerConstants.ClientTlsRedirect, StringComparison.OrdinalIgnoreCase))
```

Comment:
>if (url.StartsWith(BrokerConstants.ClientTlsRedirect, StringComparison.OrdinalIgnoreCase)) [](start = 16, length = 90)

Before moving ahead with this I would like to get the CA team perspective on us doing this. Ideally the service should not send this. 
Also not sure if the https piece would be another part of the issue or not, also to be explored.

Reply:
Synced with CA team and they approved this fix.